shared/chat/conversation/list/container.js

```diff
@@ -119,13 +119,13 @@ const mapDispatchToProps = (dispatch: Dispatch): DispatchProps => ({
     localMessageState?: Types.LocalMessageState,
     onShowEditor?: () => void,
     onPopupWillClose?: () => void,
-    targetNode?: ?HTMLElement
+    targetRect?: ?ClientRect
```

Comment:
why is this change required?

Reply:
so it'll work in native?

Reply:
@chrisnojima it was causing V8 crashes! seems we can't put HTMLElements in routeState because (probably) when some part of our stack hits the circular webContents attr of the HTMLElement everything goes to 100% CPU.

Reply:
but is this cause you have routePrinting turned on?

Reply:
It happened in a standard start-hot run.. even if it requires debugging on, would rather just keep it out of the routeState entirely, since many people do use debugging and when the crash happens there's no indication what the cause is.